content/management/configuration-api/v3.3/index.mdx

```diff
@@ -3678,6 +3678,136 @@ curl -X POST \
 
 </Section>
 
+<Section>
+
+<Text>
+
+### Enable Webhooks
+
+Enables webhooks assigned to **Client Id** from authorization token.
```

Comment:
I think the definition should be even more descriptive.

> Enables the webhooks [registered](#register-webhooks) for a given **Client Id** (application) on the account (license) associated with the access token.

WDYT?

Reply:
1. I'd use the word "registered" instead of "assigned" so that we can point to the method.
2. Do you think it will be clear which access token we're talking about?